### ISSUE TITLE
Allow passing on config file path to notmuch

### ISSUE THREAD

**Issue by rfkat** [DavidMStraub/netviel#11]
Thanks for netviel!

Since I have to handle a diverse set of maildirs (business-vs.-private issues), it would be helpful if I could pass a notmuch config file to netviel, and have it passed on to notmuch via `--config`. As a workaround, I'm symlinking the respective files to `~/.notmuch-config` before firing up netviel, which is a bit awkward.

**Comment by DavidMStraub** [comment]
Yes, good point! Are you interested in contributing the feature?

**Comment by rfkat** [comment]
A decade ago, I might have volunteered, but my development days are over since end of 2008 - have never worked on github, sorry!
